packages/react/src/dialog/index.parts.ts

```diff
@@ -2,6 +2,7 @@ export { DialogBackdrop as Backdrop } from './backdrop/DialogBackdrop';
 export { DialogClose as Close } from './close/DialogClose';
 export { DialogDescription as Description } from './description/DialogDescription';
 export { DialogPopup as Popup } from './popup/DialogPopup';
+export { Portal } from '../portal/Portal';
```

Comment:
I'm not sure if the new docs handle it differently, but I think you might need to create a DialogPortal file that exports the Portal as DialogPortal to make the API gen recognize it. I'll rework it so it recognizes the actual parts, but in the meantime, this might be the necessary workaround.

Reply:
This doesn't seem to work with "No suitable component definition found." 

`MenuSeparator` also isn't exported separately

Reply:
Looks like we need to wrap, destructure etc. in order for the current gen to work so we don't gain much from the separate Portal file anyway as of right now.

Reply:
It's still slightly better IMO now as you don't have to literally copy the component source 7 times, and will get progressively better as we fix the docgen.